src/AppBundle/Resources/views/book/edit_contribution.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Book Contribution Edit{% endblock %}
+
+{% block body %}
+    <h1>Book Contribution Edit</h1>
+
+    {{ form_start(edit_form) }}
+    {{ form_widget(edit_form) }}
+    <div class='btn-group'>
+        <input type="submit" value="Update" class="btn btn-primary" />
+        <a href="{{ path('book_show_contributions', { 'id': contribution.publication.id }) }}" class="btn btn-warning">Cancel</a>
+    </div>
+    {{ form_end(edit_form) }}
+
+{% endblock %}
```

Comment:
Do you need the styles and javascripts blocks for the form to work?

Reply:
I think it might be. The typeahead stuff shouldn't work without them.

Reply:
I tried it out on my computer and they were necessary for the form to work. Try changing an author, for example. I couldn't do it without the blocks.

Reply:
I see it now. I missed that when I was checking to see if editing worked because I was only changing the role. Getting that added in now. Thanks.